### DISCUSSION TITLE
Time filters not being applied

### DISCUSSION THREAD

**Discussion by ghost** [DNSCrypt/dnscrypt-proxy#1837] [Potential issues]
Hello.

I've downloaded  the binaries and followed the instructions in [the wiki](https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Installation-linux), so far everything works but when I try to test with the time filters they don't seem to apply from version 2.0.46-beta1 to 2.1.0.

I renamed example-dnscrypt-proxy.toml to dnscrypt-proxy.toml and example-blocked-names.txt to blocked-names.txt

In the config I just uncommented the blocked_names section and the schedules section. And in the blocked-names.txt I uncommented the youtube pattern and facebook.com.

When I test with a work schedule, I expect facebook to be blocked.

On 2.0.45

```
dig facebook.com

"This query has been locally blocked" "by dnscrypt-proxy"
```
as expected but on the versions mentionned above

```
dig facebook.com

facebook.com.           2399    IN      A       157.240.208.35
```

The blocking of a domain without a time filter works fine.

**Comment by nariox** [comment]
Same here. Maybe we should open an issue for it?

**Reply by ghost** [reply]
Yeah I just wanted to confirm it was not a me problem, I'm going to open one.

**Reply by nariox** [reply]
Yes, I have reddit set up to be blocked after 21, but it has not been blocking it since a few versions ago.

**Reply by nariox** [reply]
Oh, version 2.1.1 is just out and it seems it should fix this issue. I'll try it out.

**Reply by ghost** [reply]
Keep me updated, if I need to reopen the issue

**Reply by nariox** [reply]
Yes, seems to be fixed for me.

**Reply by ghost** [reply]
Yeah the issue is solved for me too! I'll close this
